src/zcash/Address.hpp

```diff
@@ -90,6 +90,12 @@ struct ReceiverIterator {
     size_t cur;
 };
 
+/** A recipient address to which a unified address can be resolved */
+typedef std::variant<
+    CKeyID,
+    CScriptID,
+    libzcash::SaplingPaymentAddress> RecipientAddress;
```

Comment:
Isn't this just the `RawAddress` type that I previously introduced, and you removed as unnecessary?